packages/tendermint-rpc/src/pubkeys.ts

```diff
@@ -0,0 +1,27 @@
+// These types largely overlap with the types from @cosmjs/amino
+
+export interface Pubkey {
+  // type is one of the strings defined in pubkeyType
+  // I don't use a string literal union here as that makes trouble with json test data:
+  // https://github.com/cosmos/cosmjs/pull/44#pullrequestreview-353280504
+  readonly type: string;
+  readonly value: any;
+}
+
+export interface Ed25519Pubkey extends Pubkey {
+  readonly type: "tendermint/PubKeyEd25519";
```

Comment:
Those type identifiers are Amino specific and not needed anymore 🎉 The pubkeys are now just defined as Ed25519 or Secp256k1: https://github.com/tendermint/tendermint/blob/v0.34.8/proto/tendermint/crypto/keys.proto#L8-L17

This means we can use a simpler API here, like:

```
export function pubkeyToRawAddress(type: "ed25519" | "secp256k1", data: Uint8Array): Uint8Array
```

Then users can come with all their different JSON dumps or ptotobuf files and bring it in this simple form.

Sorry for pretending that reusing the Amino `Pubkey` types is a good idea.